kyo-core/shared/src/test/scala/kyoTest/typeMapTest.scala

```diff
@@ -157,14 +158,44 @@ class typeMapTest extends KyoTest:
     }
 
     ".prune" - {
-        "Env[Int & String] -> Env[Int]" in pendingUntilFixed {
-            assertCompiles(
-                """
-                  | val e = TypeMap(42, "")
-                  | val p = e.prune[Int]
-                  | assert(p.size == 1)
-                  |""".stripMargin
-            )
+        "Env[Int] -> Env[Any]" in {
```

Comment:
Can you add type annotations to the values in the tests? It's not clear to me what's expected.

Reply:
If you don't expect the annotations to change the behavior of the API, it could be all tests but just a couple of them should be enough to make it easier to understand the semantics of the API.